examples/with-hasura-thirdpartyemailpassword/api-server.js

```diff
@@ -52,11 +52,18 @@ supertokens.init({
             ],
         }),
         Session.init({
-            jwt: {
-                enable: true,
-                issuer: process.env.JWT_ISSUER_URL,
-            },
+            getTokenTransferMethod: () => "header",
```

Comment:
Why did this change?

```suggestion
            exposeAccessTokenToFrontendInCookieBasedAuth: true,
```